### PR TITLE
Add validation conditions to app summary

### PR DESCRIPTION
**Pull request by orangejenny**
https://manage.dimagi.com/default.asp?251826

@emord

### CODE REVIEW

File: corehq/apps/app_manager/xform.py [hunk 1]
```diff
@@ -954,7 +954,7 @@ def get_questions(self, langs, include_triggers=False,
         control_nodes = self.get_control_nodes()
         leaf_data_nodes = self.get_leaf_data_nodes()
 
-        for node, path, repeat, group, items, is_leaf, data_type, relevant, required in control_nodes:
+        for node, path, repeat, group, items, is_leaf, data_type, relevant, required, constraint in control_nodes:
```
Comment (emord) [inline corehq/apps/app_manager/xform.py]: This is easily the most values i've seen in a for loop. (not that you need to fix that here)

**Review by emord — APPROVED** [review]

File: corehq/apps/app_manager/xform.py (hunk 1 above)
Comment (orangejenny) [inline corehq/apps/app_manager/xform.py]: Yeah, it's a bit much. Switched to a namedtuple, which is at least a little bit more readable.

**Comment by proteusvacuum** [comment]
Hey, 
Think it would be worthwhile to add these to the download too?

**Comment by orangejenny** [comment]
@proteusvacuum adding them to `get_questions` adds them to the download, too

**Comment by proteusvacuum** [comment]
@orangejenny oh, cool. I thought you would have needed to define an extra column in the excel sheet, but maybe not?

**Comment by orangejenny** [comment]
haven't looked at the code...maybe the excel sheet config would let me do things like rename the column? but it does work, adds a `relevant` column.

**Comment by orangejenny** [comment]
Offlined with Farid; I was being dumb. Validation conditions should be in download for real now.

**Pull request merged**